src/BlazorUI/Bit.BlazorUI/Components/Progress/Shimmer/BitShimmer.scss

```
@@ -3,6 +3,16 @@
 .bit-smr {
     display: block;
     --bit-smr-bg-clr: #{$clr-bg-ter};
+    // The sizing properties are inherited, and a Template is built out of shimmers of its own - so a height,
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)